src/main/resources/config/liquibase/changelog/20240418201711_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240418201711" author="mkawka">
+        <addColumn tableName="exam_live_event">
+            <column name="problem_statement" type="longtext"/>
+            <column name="exercise_id" type="bigint"/>
```

Comment:
You should probably add a foreign key for this column

Reply:
done